transform/mattermost-analytics/models/marts/product/rpt_tedau_at_day_28.sql

```diff
@@ -0,0 +1,19 @@
+  with retention_at_28 as (
+    select
+        fct.server_id,
+        fct.daily_active_users,
+        fct.server_daily_active_users
+    from
+        {{ ref('fct_active_users') }} fct
+        join {{ ref('dim_daily_server_info') }} dsi on fct.daily_server_id = dsi.daily_server_id and dsi.age_in_days = 28
+        left join {{ ref('dim_server_info') }} si on fct.server_id = si.server_id
```

Comment:
just asking Yiannis, why we'd need this extra join? Thanks.

Reply:
It's not needed. Thanks for noticing! It's now removed.